From: Pablo Neira Ayuso <pablo@netfilter.org>
Date: Sat, 21 Feb 2009 18:13:06 +0000 (+0100)
Subject: src: fix compilation issue in gentoo due to missing include limits.h
X-Git-Url: https://git.netfilter.org/cgi-bin/gitweb.cgi?p=conntrack-tools.git;a=commitdiff_plain;h=9bf002ff7935e7dff625683787fc3a06ac2ef2cb;hp=87c8c090e1b64eac27edb968845cb1e8ee65b06b

src: fix compilation issue in gentoo due to missing include limits.h

This patch adds include limits.h to mcast.c and sync-mode.c. Why Gentoo
maintainers did not report me the problem? :(

http://bugs.gentoo.org/show_bug.cgi?id=256497

Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
---

diff --git a/src/mcast.c b/src/mcast.c
index ba472f6..8f11762 100644
--- a/src/mcast.c
+++ b/src/mcast.c
@@ -28,6 +28,7 @@
 #include <sys/ioctl.h>
 #include <net/if.h>
 #include <errno.h>
+#include <limits.h>
 #include <libnfnetlink/libnfnetlink.h>
 
 struct mcast_sock *mcast_server_create(struct mcast_conf *conf)
diff --git a/src/sync-mode.c b/src/sync-mode.c
index d1a941b..26e1358 100644
--- a/src/sync-mode.c
+++ b/src/sync-mode.c
@@ -32,6 +32,7 @@
 #include <time.h>
 #include <string.h>
 #include <stdlib.h>
+#include <limits.h>
 #include <net/if.h>
 
 static void
